src/status_im2/contexts/syncing/how_to_pair/view.cljs

```diff
@@ -49,7 +49,8 @@
      (i18n/label (:label value))]))
 
 (defn render-item
-  [i item]
+  [i list-item]
+  ^{:key i}
```

Comment:
Additionally, Added the key to the iterations as the metro bundler throws a lot of warnings.